config/resources/lpdc.json

```diff
@@ -0,0 +1,644 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "adms": "http://www.w3.org/ns/adms#",
+    "code": "http://lblod.data.gift/concepts/public-services/",
+    "cspv": "http://purl.org/vocab/cpsv#",
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "dct": "http://purl.org/dc/terms/",
+    "foaf": "http://xmlns.com/foaf/0.1/",
+    "locn": "http://www.w3.org/ns/locn#",
+    "lpdcExt": "http://lblod.data.gift/vocabularies/lpdcExt",
+    "m8g": "http://data.europa.eu/m8g/",
+    "prov": "http://www.w3.org/ns/prov#",
+    "ps": "http://vocab.belgif.be/ns/publicservice#",
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "schema": "http://schema.org/",
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "vcard": "http://www.w3.org/2006/vcard/ns#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "eli": "http://data.europa.eu/eli/ontology#"
+  },
+  "resources": {
+    "conceptual-public-services": {
+      "class": "lpdcExt:ConceptualPublicService",
+      "attributes": {
+        "name": {
+          "type": "string",
+          "predicate": "dct:title"
+        },
+        "description": {
+          "type": "string",
+          "predicate": "dct:description"
+        },
+        "keyword": {
+          "type": "string",
+          "predicate": "dcat:keyword"
+        },
+        "moreInfo": {
+          "type": "string",
+          "predicate": "lpdcExt:moreInfo"
+        },
+        "regulations": {
+          "type": "string",
+          "predicate": "lpdcExt:regulation"
+        },
+        "exceptions": {
+          "type": "string",
+          "predicate": "lpdcExt:exception"
+        },
+        "start-date": {
+          "type": "datetime",
+          "predicate": "schema:startDate"
+        },
+        "end-date": {
+          "type": "datetime",
+          "predicate": "schema:endDate"
+        },
+        "created": {
+          "type": "datetime",
+          "predicate": "dct:created"
+        },
+        "modified": {
+          "type": "datetime",
+          "predicate": "dct:modified"
+        },
+        "language": {
+          "type" : "string",
+          "predicate": "dct:language"
+        }
+      },
+      "relationships": {
+        "identifier": {
+          "target": "identifiers",
+          "predicate": "dct:identifier",
+          "cardinality": "one"
+        },
+        "related": {
+          "target": "conceptual-public-services",
+          "predicate": "dct:relation",
+          "cardinality": "many"
+        },
+        "hasInstance": {
```

Comment:
I think some of these relationship names are a bit weird. They will be used in the frontend as defined here, so for example we would have to use `conceptualPublicServiceRecord.hasInstance` to receive a list of services which isn't intuitive when reading the code 😄. `instances` might be a more intuitive name, maybe?

I would try to give all relationships a more logical sounding name, especially the ones with "many" cardinality.

Reply:
Sure, I can drop the `has*` from the json api. It's now literally taken from the rdf properties.